File: server/src/main/java/com/server/model/Menu.java
package com.server.model;

import lombok.Data;

@Data
public class Menu {
    private int id;
    private String Name_zh_HK;
    private String Name_zh_CN;
    private String Name_en_US;
    private String onSale;
    private int price;
    private int type;
    private String path;
}

File: customer/src/api/Menu.js
import axios from "axios";

const baseURL = "http://localhost:8080/api/menu";

export function findInMenu(
  name_zh_HK,
  name_zh_CN,
  name_en_US,
  price,
  onSale,
  type
) {
  return axios.post(baseURL + "/findInMenu", {
    name_zh_HK,
    name_zh_CN,
    name_en_US,
    price,
    onSale,
    type,
  });
}
